doc/classes/StyleBoxLine.xml

```diff
@@ -1,23 +1,30 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="StyleBoxLine" inherits="StyleBox" category="Core" version="3.2">
 	<brief_description>
+		StyleBox that displays a single line.
```

Comment:
Same here, you can use `[StyleBox]` and `[StyleBoxLine]` (for the Self class, the makerst script will automatically convert it to inline code formatting instead of making a useless hyperlink to Self).

Reply:
Bump.